
=== Batch Execution Failure Report ===
Generated at: 2025-07-13T00:11:47.158021016

Experiment: scalability_2000_200
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_1000_100
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_1000_200
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_500_100
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: sensitivity_max_iterations
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_5000_500
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_1000_500
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_2000_500
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: baseline_comparison_AntColony
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_100_100
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_100_50
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_100_500
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_500_200
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_100_10
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_100_200
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_500_500
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: baseline_comparison_GeneticAlgorithm
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: scalability_500_50
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: real_dataset_azure_traces
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: real_dataset_google_traces
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: baseline_comparison_HippopotamusOptimization
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


Experiment: baseline_comparison_ParticleSwarm
Error Type: org.cloudbus.cloudsim.experiment.ExperimentRunner$ExperimentException
Error Message: Experiment execution failed
Stack Trace: 	at org.cloudbus.cloudsim.experiment.ExperimentRunner.runExperiment(ExperimentRunner.java:91)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.executeExperimentWithRetry(BatchExperimentExecutor.java:290)
	at org.cloudbus.cloudsim.experiment.BatchExperimentExecutor.lambda$parallelExecution$1(BatchExperimentExecutor.java:208)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)


